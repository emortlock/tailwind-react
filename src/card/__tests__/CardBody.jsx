import React from 'react'
import { shallow } from 'enzyme'

import { CardBody } from '..'

const setup = (testProps = {}) => {
  const props = Object.assign(testProps)

  const wrapper = shallow(<CardBody {...props}>Hello World!</CardBody>)

  return {
    props,
    wrapper,
  }
}

describe('CardBody', () => {
  it('renders matching snapshot', () => {
    const { wrapper } = setup()

    expect(wrapper).toMatchSnapshot()
  })
})
