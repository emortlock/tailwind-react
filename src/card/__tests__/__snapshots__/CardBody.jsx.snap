// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBody renders matching snapshot 1`] = `
<div
  className="p-4"
>
  Hello World!
</div>
`;
